packages/parse5-test-utils/data/parser-feedback/tests6.test

```diff
@@ -134,8 +134,8 @@
             ]
         },
         {
```

Comment:
Later

Reply:
Also: why is `\r` no longer supported?!

Reply:
They were removed by the script. Seems like a change in the upstream test data that this is derived from.

Reply:
weird